server.py

```diff
@@ -61,8 +64,10 @@ def leaderboard():
                 + user["hard"] * POINTS["hard"]
             )
         users_sorted = sorted(users, key=lambda u: u["points"], reverse=True)
+        MetricsHandler.sign_last_updated.set(int(time.time()))
         return users_sorted
     except Exception as e:
+        MetricsHandler.sign_update_errors.inc()
```

Comment:
can this instead be set to a 0 when its fine, and a 1 in here?

that way we can alert for when this metric has been 1 for a certain amount of time